tests/test_world.py

```diff
@@ -0,0 +1,253 @@
+import random
+
+import pytest
+
+from tempus_fugit_minecraft.block import (BRICK, DARK_CLOUD, GRASS,
```

Comment:
Same suggestion as above, I recommended changing to reduce total line length:
```
from tempus_fugit_minecraft.block import (BRICK, DARK_CLOUD, GRASS, LIGHT_CLOUD, SAND, STONE, TREE_LEAVES, TREE_TRUNK, 
                                          Block)
```

Reply:
Done